src/viam/api/CMakeLists.txt

```diff
@@ -230,9 +244,7 @@ if (VIAMCPPSDK_USE_DYNAMIC_PROTOS)
     ${PROTO_GEN_DIR}/tagger/v1/tagger.pb.cc
     ${PROTO_GEN_DIR}/tagger/v1/tagger.pb.h
 
-    COMMAND ${BUF_COMMAND} generate ${BUF_GOOGLE_API_SOURCE} --template buf.gen.yaml --path google/rpc --path google/api
-    COMMAND ${BUF_COMMAND} generate ${BUF_VIAM_GOUTILS_SOURCE} --template buf.gen.yaml
-    COMMAND ${BUF_COMMAND} generate ${BUF_VIAM_API_SOURCE} --template buf.gen.yaml --path ${BUF_PROTO_COMPONENTS_JOINED}
+    ${CMAKE_COMMAND} -P invoke_buf_generate.cmake
```

Comment:
Does this `add_custom_command` need a dependency edge to the generated `invoke_buf_generate.cmake`, like it has to the `buf.gen.yaml`?

Reply:
I think so but also I slept on it and realized I can just pass stuff to the script with `cmake -DSOME_ARG=VAL -P script.cmake` rather than using `configure_file` on `script.cmake.in` which I think is preferable for not introducing more state.

Reply:
Update: this turned out not to be necessary, see documentation in BUILDING.md about setting the environment variables.